prog/vm/github_runner.rb

```diff
@@ -55,6 +59,23 @@ def github_client
     @github_client ||= Github.installation_client(github_runner.installation.installation_id)
   end
 
+  def self.extract_vm_features(label)
```

Comment:
How about simple lookup table instead? It would be easier to read & maintain. It also has nice benefit of being a place where we can see all the labels we support.

Reply:
Thanks. It makes easier to define valid vm sizes, locations, boot_images for runners. Also code looks cleaner now